tensorflow/core/kernels/data/experimental/assert_next_dataset_op.h

```diff
@@ -21,8 +21,8 @@ namespace tensorflow {
 namespace data {
 namespace experimental {
 
-// See documentation in ../../ops/experimental_dataset_ops.cc for a high-level
-// description of the following op.
+// See documentation in tensorflow/core/ops/experimental_dataset_ops.cc for a
```

Comment:
[The relative path](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/kernels/data/batch_dataset_op.cc#L29) is used in for the dataset ops under `tensorflow/core/kernels/data/`. Do we need to keep them consistent? cc: @jsimsa

Reply:
My suggestion would be to simply get rid of all of these comments because the `experimental_dataset_ops.cc` and `dataset_ops.cc` files no longer contain documentation.

Reply:
I can certainly remove the comments if that's best.

Should I put in pointers to docstrings in Python files where a relevant docstring exists?

Reply:
The Python API and the C++ API are not the same, so I am not sure it makes sense to do that.

One thing we could do (likely over multiple PRs) is to update the api_def_* files (e.g. https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/api_def/base_api/api_def_AssertNextDataset.pbtxt) to documentation the op API and then to point to those from the C++ files.

If you would like to take this on, I would be happy to review the PRs. If not, simply remove the comments.

Reply:
Sure, I'd be happy to take on updating the API def files as a follow-on task. I'll modify the current PR so it just removes the outdated comments.